components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-overview/overview-trend/overview-trend.component.ts

```diff
@@ -51,9 +52,13 @@ export class OverviewTrendComponent implements OnChanges, OnDestroy  {
   }
 
   get trendData(): TrendData[] {
-    return this.data.map(d => {
-      return { ...d, report_time: this.createUtcDate(d.report_time) };
-    });
+    if(this.treandDataCache === undefined || this.treandDataCache.length == 0) {
+      console.log("trendDataCache..!")
+      this.treandDataCache = this.data.map(d => {
+        return { ...d, report_time: this.createUtcDate(d.report_time) };
+      });
+    }
+    return this.treandDataCache
```

Comment:
missing semicolon
```suggestion
    return this.treandDataCache;
```

running `make lint` in your terminal while in `components/automate-ui` will do a full lint check and show you any linting issues to fix